golem/payload/generic.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from .base import Constraints, Payload, Properties
+from .parser import PayloadSyntaxParser
```

Comment:
Why `.` imports?

Reply:
why not? it's more portable and less verbose...

Reply:
anyway, updated